src/introduction/abtnode-overview/index.md

```diff
@@ -9,42 +9,55 @@ tags:
+An ArcBlock instance is composed of one or more ABT Nodes. Each ABT Node offers the ability to create decentralized applications (DApps). The ABT Nodes offer a fully integrated blockchain framework with decentralized identity while having a  small footprint. It can be deployed on diverse platforms like a public or private clouds, bare metal machines, or developer desktops. ABT  Nodes offers intuitive  UI to build blockchain nodes, smart contracts, application logic, and more with zero hassle. Users with ABT Wallet can access the DApps deployed on the ABT Nodes.
 
-## Dashboard
+![](./images/abtnode-arch-overview-en.png)
 
-In the dashboard view, you are presented with a "single pane of glass" view that shows you key details including an overview, storage, your node and owner decentralized identity (DID) addresses and environment variables. Every node also includes a left side action bar where that let you interact and control different aspects of your ABT Node. Let's take a look:
+In the following actions, we will go over all the capabilities offered by the ABT Node in detail.
 
-![](./images/abtnode-overview-1-en.png)
+## ABT Dashboard
 
-Dashboard of ABT node state properties.
+The dashboard offers a unified view of the underlying infrastructure available with the ABT Node. It lists down different tenets of ABT Node, not limited to the following :
+- Storage
+- CPU
+- Memory
+- Disk
+- UpTime
+- Environment
+
+![](./images/abtnode-dashboard-en.png)
 
 ## Marketplace
 
-![](./images/abtnode-overview-2-en.png)
+Blocklets are the foundation block for your application. They can deliver various behaviours like DApps, smart contracts, blockchain etc. The Marketplace is a digital catalog of Blocklets from ArcBlock, ecosystem partners, and software vendors that make it easy to find, install, manage and one-click deploy Blocklets on your ABT Node.
 
-Your ABT Node Marketplace is a digital catalog of Blocklets from ArcBlock, ecosystem partners and software vendors that make it easy to find, install, manage and one-click deploy Blocklets on your ABT Node.
+![](./images/abtnode-marketplace-en.png)
 
 ## Blocklets
 
-![](./images/abtnode-overview-3-en.png)
+The Blocklet management section shows you all of the Blocklets installed on your ABT Node. You can perform various actions like start, stop, reload and remove individual Blocklets. You can also access each Blocklet to review overview information and configuration details needed for development projects.
```

Comment:
blocklet configurations are not only used for development purpose, it's most used to customize the behavior of the blocklet at runtime.

Reply:
reworded the section